src/services/Teambuilding.ts

```diff
@@ -166,11 +167,11 @@ function findTeamAssignmentByLearnerId(
  */
 export async function processLabsApplication(
   labsApplicationSubmission: ILabsApplicationSubmission
-): Promise<void> {
+): Promise<IFinalLabsProject> {
   const oktaId = labsApplicationSubmission.oktaId;
   const labsApplication = labsApplicationSubmission;
-  console.log(labsApplicationSubmission)
-  try { 
+  console.log(labsApplicationSubmission);
+  try {
     // Parse the learner's GitHub handle as a profile URL
     const gitHubUrl = await buildGitHubUrl(labsApplication.gitHubHandle || "");
     // Get the learner's Salesforce Contact ID by their OktaID
```

Comment:
Sadly, I rabbit-holed on this. I thought the contactId is on the JDS enrollment object as well but I couldn't find the link to the object explorer to confirm. (got it in zoom chat only)

Reply:
We could definitely avoid some calls based on where this landed—we could be getting the Contact ID in a query for the JDS Track Enrollment object via `JDS_Track_Enrollment__r.Contact__r.Id` and avoid this extra round trip. I think we should do this cleanup in a subsequent release.